### DISCUSSION TITLE
Using visio in github

### DISCUSSION THREAD

**Discussion by carlblunck** [community/community#138337] [Programming Help]
### Select Topic Area

Question

### Body

Hi all,

Have only just started using github for tracking sql query changes and I want to also use it for tracking changes to my data architecture models that I have in visio.

It seems this is possible but I have no idea how to add something like this - https://github.com/YerongAI/Office-Tool 

If someone could help with step by step instructions on what to do that would be amazing!!

Thanks
Carl

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @carlblunck ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
